app/models/goal.py

```diff
@@ -1,6 +1,14 @@
 from flask import current_app
 from app import db
-
+from app.models.task import Task
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", backref="task", lazy=True)
+
+    def goal_json(self):
+        return( {
+            "id": self.goal_id,
+            "title": self.title
+        }, 200)
```

Comment:
This method is the main reason wave 5 has some failures. Your routes logic is trying to call a method called `goal_to_json` but the method here is called `goal_json`. Also, we should only be returning the dictionary here, not the status code.

The following change would allow Wave 5 to pass!

```suggestion
    def goal_to_json(self):
        return {
            "id": self.goal_id,
            "title": self.title
        }
```